### DISCUSSION TITLE
ご意見のページのリンクにベータテストの表記

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#124] [要望・アイデア・不具合]
細かい話ですが、[ご意見のページ](https://qiita.com/feedback/new)に、ここGitHub Discussionsへのリンクがありますが、まだ「GitHub Discussionsを用いてベータテストを実施しています。」と、ベータテストの表記が残っているようです。

**Comment by kskwtnk** [comment]
@yoshi389111 
ありがとうございます！こちらすぐにアップデートいたします！！

**Comment by kyntk** [comment]
@yoshi389111 ベータテストの表記の削除完了いたしました!
ご報告ありがとうございました!

**Comment by yoshi389111** [comment]
クローズした物に追記してすみません、今気が付いたのですが [CODE_OF_CONDUCT](https://github.com/increments/qiita-discussions/blob/main/CODE_OF_CONDUCT.md) にもパブリックベータの記載が残っているようです。

**Comment by kyntk** [comment]
@yoshi389111 ありがとうございます! 対応いたします! 🙇🏽

**Comment by kskwtnk** [comment]
@yoshi389111 
こちらご指摘ありがとうございました！

https://github.com/increments/qiita-discussions/blob/main/CODE_OF_CONDUCT.md

アップデート完了いたしました！
